lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -74,11 +75,6 @@ export class SilentIframeClient extends StandardInteractionClient {
             PerformanceEvents.SilentIframeClientAcquireToken,
             request.correlationId
         );
-        this.logger.verbose("acquireTokenByIframe called");
```

Comment:
Did you intend to remove this logger line as well?

Reply:
Yes, the invoke functions will handle logging start/end messages as well